test/JDBC/input/views/sys-database_mirroring-vu-prepare.sql

```diff
@@ -0,0 +1,15 @@
+CREATE VIEW sys_database_mirroring_view AS
```

Comment:
please use name like sys_database_mirroring_vu_prepare_view, this is consistent as it uses file name as prefix and will not conflict with non-uprgrade tests. Applies to all object created in vu-prepare scripts.

Reply:
Sounds good, will do!